script/Arcueid/Day 5/05_00_ARC05_1C.txt

```diff
@@ -323,12 +324,12 @@ If you were to compare my Lifescale with Vlov's right now, he would be slightly
 [sha:e8f11727089cfd7149d6876d813674b4ae448da2]{
 -- Page 168, Offset 8994.
 -- 普段のアルクェイドなら、なんとか倒せる相手なんだ？」
-So if you were in your normal condition, you would be able to defeat him somehow, right?"
+So if you were at peak performance, you'd be able to defeat him somehow, right?"
```

Comment:
revert "normal condition"

Reply:
You may not like it but this is what normal condition looks like